seml/parameters.py

```diff
@@ -163,8 +161,7 @@ def generate_grid(parameter, parent_key=''):
 
     """
     if "type" not in parameter:
-        logging.error("No type found in parameter {}".format(parameter))
-        sys.exit(1)
+        raise ConfigError("No type found in parameter {}".format(parameter))
```

Comment:
Very minor point: usually we use f-strings.

Reply:
Fixed in 6274e6c29b6f0f41c2597dee63188f394f212413.